src/tokens/eth/0x0583fb65766351B9202091Ba695c702808015183.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
this token does not exist under the eth network
make sure to place the file in the correct network
https://bscscan.com/token/0x0583fb65766351b9202091ba695c702808015183